test/helpers/test_qiskit_to_ionq.py

```diff
@@ -206,6 +206,58 @@ def test_full_circuit(simulator_backend):
     assert actual == expected_rest_of_payload
 
 
+def test_multicircuit_mapping(simulator_backend):
+    """Test multiple circuits with mapped qubits
+
+    Args:
+        simulator_backend (IonQSimulatorBackend): A simulator backend fixture.
+    """
+    no_reg_map = QuantumCircuit(2, 2, name="no_reg_map")
+    no_reg_map.x(0)
+    # measure 0 to 1 and 1 to 0
+    no_reg_map.measure([0, 1], [1, 0])
+    no_reg_no_map = QuantumCircuit(2, 2, name="no_reg_no_map")
+    no_reg_no_map.x(0)
+    # measure 0 to 0 and 1 to 1
+    no_reg_no_map.measure([0, 1], [0, 1])
+
+    ionq_json = qiskit_to_ionq(
+        [no_reg_map, no_reg_no_map],
+        simulator_backend,
+        passed_args={"shots": 1024},
+    )
+
+    expected_payload = {
+        "target": "simulator",
+        "shots": 1024,
+        "name": "no_reg_map, no_reg_no_map",
+        "input": {
+            "format": "ionq.circuit.v0",
+            "gateset": "qis",
+            "qubits": 2,
+            "circuits": [
+                {
+                    "circuit": [{"gate": "x", "targets": [0]}],
+                    "registers": {"meas_mapped": [1, 0]},
+                },
+                {
+                    "circuit": [{"gate": "x", "targets": [0]}],
+                    "registers": {"meas_mapped": [0, 1]},
+                },
+            ],
+        },
+        "metadata": {
+            "shots": "1024",
+            "sampler_seed": "None",
+        },
+        "noise": {"model": "ideal", "seed": None},
+    }
+
+    actual = json.loads(ionq_json)
+    actual["metadata"].pop("qiskit_header")  # remove qiskit_header for comparison
+    assert actual == expected_payload
```

Comment:
any way to test the flip-side? On a map with particular values, that the results are mapped correctly?

Reply:
oh ya let me add that